components/brave_rewards/resources/page/ping_doc_signer/components/Header/Header.tsx

```diff
@@ -92,13 +98,21 @@ export const Header: React.FC<HeaderProps> = ({
                 <StyledFadeAway fadeAnimation={isStatusVisible}>
                     <StyledHeaderButton pdfFile={pdfFile} onClick={handleSignButtonClick}>Add signature</StyledHeaderButton>
                     <StyledHeaderControlsBar />
-                    <StyledHeaderButton
-                        pdfFile={pdfFile}
-                        onClick={handleVerifyButtonClick}
-                        as={isVerified ? StyledVerified : isVerificationFailed ? StyledNotVerified : 'button'}
+                    <Tooltip
+                        text="Verification failed. Please check the document."
+                        isVisible={showVerificationTooltip && isVerificationFailed}
+                        isError={true}
                     >
-                        Verify document
-                    </StyledHeaderButton>
+                        <StyledHeaderButton
+                            pdfFile={pdfFile}
+                            onClick={handleVerifyButtonClick}
+                            as={isVerified ? StyledVerified : isVerificationFailed ? StyledNotVerified : 'button'}
+                            onMouseEnter={() => setShowVerificationTooltip(true)}
```

Comment:
we don't need to use mouse events to show/hide tooltip. we could have also used css hover properties to show hide tooltip. otherwise we would have to create so many states. 

But we can go with this for now as well

Reply:
Or we could move these states to the tooltip component as well. that would be easy and clean up the code